src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 17px;
+  padding-top: 33px;
+  padding-bottom: 18px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 30px;
+  }
+
+  &__title {
+    justify-content: center;
```

Comment:
The `justify-content` property is used with flex containers and it won't have any effect on non-flex elements. Since `.card__title` is not a flex container itself (it does not have `display: flex`), this property is not needed here. If you intended to center the text, use `text-align: center;` instead.